main.js

```diff
@@ -502,6 +502,14 @@ exports.main = function (options, callbacks) {
   // Override default preferences
   utils.setDefaultPrefs();
 
+  browserWindows.on('close', function (window) {
+    let win = viewFor(window);
+    let mainWindow = win.document.getElementById('main-window');
+    if (mainWindow.getAttribute('toggledon') === 'true') {
```

Comment:
Should we also check for `win.VerticalTabs`?  I mean, I don't think it won't exist ever here, but just in case?